lib/components/SDescNumber.vue

```diff
@@ -6,6 +6,8 @@ import SDescEmpty from './SDescEmpty.vue'
 
 const props = defineProps<{
   value?: number | null
+  leadUnit?: string
+  trailUnit?: string
```

Comment:
https://github.com/globalbrain/sefirot/blob/bd886cb78cbe38228f16e85c9a27a06a92828675/lib/components/SInputNumber.vue#L21-L22

https://github.com/globalbrain/sefirot/blob/bd886cb78cbe38228f16e85c9a27a06a92828675/lib/components/SInputText.vue#L138-L141

You might want to call the props `unitBefore/unitAfter` for consistency?

Reply:
Sorry forgot to mention, since `<SActionList>`, it's changed to lead/trail following GitHub primer 👀